pod/perlfunc.pod

```diff
@@ -6536,6 +6536,11 @@ the C<\Q> escape in double-quoted strings.
 
 If EXPR is omitted, uses L<C<$_>|perlvar/$_>.
 
+The motivation behind this is to make all characters in EXPR have their
+literal meanings.  Otherwise any metacharacters in it could trigger
+their "magic" actions.  The characters are said to be "quoted" or
```

Comment:
I don't think metachacters have "actions", maybe they "behave", but the definitely don't "act"

Reply:
changed as suggested